<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListWidget" name="list_vertexes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>260</width>
      <height>465</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>260</width>
      <height>465</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>260</width>
      <height>465</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="button_readFile">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>115</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>115</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>115</width>
      <height>25</height>
     </size>
    </property>
    <property name="text">
     <string>Read</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_openFile">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>115</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_fileName">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>115</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>25</height>
    </size>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
